src/coreComponents/linearAlgebra/interfaces/hypre/HypreMatrix.cpp

```diff
@@ -214,6 +216,7 @@ void HypreMatrix::create( CRSMatrixView< real64 const, globalIndex const > const
   localIndex const * const pSizes = localMatrix.getSizes();
   localIndex const * const pOffsets = localMatrix.getOffsets();
 
+  // Note: this relies on
```

Comment:
I got distracted and didn't finish it :) 
I was going to say that the following block relies on the fact that `localMatrix` has had its sparsity pattern built on the host, because it doesn't explicitly move sizes/offsets to `MemorySpace::CPU` before accessing them there. I will finish the comment.